PIL/ImageOps.py

```diff
@@ -178,6 +178,27 @@ def crop(image, border=0):
         )
 
 
+def scale(self, factor, resample=Image.NEAREST):
```

Comment:
We should. I use this default value to be coherent with the Image.resize function. For me, all resample default parameter should be Bicubic or bilinear